src/wast-parser.cc

```diff
@@ -1545,13 +1602,23 @@ Result WastParser::ParseConst(Const* const_) {
       result = ParseDouble(literal.type, s, end, &const_->f64_bits);
       break;
 
+    case Opcode::V128Const:
+      const_->type = Type::V128;
+      // Parse V128 Simd Const (16 bytes).
+      result = ParseSimdConst(const_, in_type, sizeof(v128));
```

Comment:
Since this is always going to be `size(v128)` for now, probably best to just hard code it in `ParseSimdConst`.

Reply:
The reason here is for future possible 256, 512 simd type too.